### DISCUSSION TITLE
Problem cmake and onnxruntime-gpu

### DISCUSSION THREAD

**Discussion by jmmsalvadorVideologicanalytics** [microsoft/vcpkg#27756] [Q&A]
Hi, I am working with vcpkg, cmake and onnxruntime-gpu

onnxruntime-gpu is installed successfully with the command
vcpkg install onnxruntime-gpu:x64-windows
https://github.com/microsoft/vcpkg/blob/master/ports/onnxruntime-gpu/vcpkg.json

In the file cmakelists.txt, I can find the includes with the line
include_directories(${ONNXRuntime-gpu_INCLUDE_DIRS})

But I can't find the library with the line
find_package(ONNXRuntime-gpu REQUIRED)

Could not find a package configuration file provided by "ONNXRuntime-gpu" with any of the following names:
 ONNXRuntime-gpuConfig.cmake
 onnxruntime-gpu-config.cmake

Is there another solution?

**Comment by JackBoosY** [comment]
Ah, the gpu version seems have some bugs, please open an issue instead.
